python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -169,7 +169,9 @@ def add_historical_run(
         )
         return dagster_run
 
-    def handle_run_event(self, run_id: str, event: DagsterEvent) -> None:
+    def handle_run_event(
+        self, run_id: str, event: DagsterEvent, update_timestamp: Optional[datetime] = None
+    ) -> None:
```

Comment:
will have to update the ABC

https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/_core/storage/runs/base.py#L57-L64

theoretically could break custom run storages, not sure how much to worry about that